### PR TITLE
fix: Filename issue while editing a macro with attachment action

### LINKED ISSUE #5732: “Send Attachment” action - UI issues
The “Send Attachment” action shows the “Upload Attachment” option in the edit screen even though the file was uploaded previously.

- Fix the action behavior
- Fix the spelling mistake (Update it to `successfully`)

Loom: https://www.loom.com/share/fbb1cb41892f45c28e70cc3dc18cb769

### PR DESCRIPTION
**Pull request by fayazara**
## Description

File name is now displayed for attachment action in macros
Fixes #5732

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Review by iamsivin — APPROVED** [review]
LGTM

File: app/javascript/dashboard/routes/dashboard/settings/macros/MacroNodes.vue [hunk 1]
```diff
@@ -61,6 +68,16 @@ export default {
...
+  methods: {
+    fileName(id, actionType) {
+      if (!id || !this.files) return '';
+      if (actionType === 'send_attachment') {
+        const file = this.files.find(item => item.blob_id === id);
+        if (file) return file.filename.toString();
+      }
+      return '';
+    },
+  },
```
[the remark this answers is not in the document]
Comment (fayazara) [reply app/javascript/dashboard/routes/dashboard/settings/macros/MacroNodes.vue]: @pranavrajs Updated, please have a look

**Comment by pranavrajs** [comment]
@muhsin-k / @iamsivin Can you please add a Loom video if you have tested the PR?

**Comment by muhsin-k** [comment]
https://www.loom.com/share/58a48c0dc3a64a88bc2f6c8736147fa0

**Pull request merged**